src/test/configs/image-with-local-feature/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Thanks for adding the test. It doesn't look like it's being executed anywhere?

Reply:
Test added for `image-with-local-feature`. I'm afraid the resulting image doesn't contain any useful details about the mount from the local feature (or if the var replacement happened). I might have to test this using cli up instead if this test is not sufficient.

Reply:
That would be great - we have plenty of [examples](https://github.com/devcontainers/cli/blob/main/src/test/container-features/e2e.test.ts) of using `up` with Features.  

Perhaps you could use the new `${remoteUser}` variable in a Feature lifecycle hook to write that in a file (and assert it as you expect)?

Reply:
If you decide to go that route, the tests in [this file](https://github.com/devcontainers/cli/blob/main/src/test/container-features/lifecycleHooks.test.ts) may be more helpful.

Reply:
Gotcha! I thought about using it in a mount directive, starting the container and checking it with `docker inspect` if the mount is used properly. But your idea is better. I'll adapt it tomorrow.

Reply:
Test adapted. Feature now writes `${remoteUser}` into `/tmp/variable-substitution.testMarker` and asserts the content.